rust/repositories.bzl

```diff
@@ -392,7 +418,14 @@ def _rust_toolchain_repository_impl(ctx):
     if ctx.attr.rustfmt_version:
         BUILD_components.append(_load_rustfmt(ctx))
 
-    for target_triple in [ctx.attr.exec_triple] + ctx.attr.extra_target_triples:
+    for target_triple in [ctx.attr.exec_triple]:
+        BUILD_components.append(_load_rust_stdlib(ctx, target_triple))
+        if ctx.attr.dev_components:
+            # No BUILD file is needed because components will match the stdlib
+            # BUILD file globs
```

Comment:
What does that comment means? It is probably not needed.

Reply:
Okay, I'll remove it. The comment was explaining why the `_load_rustc_dev_nightly` call didn't follow the pattern of being inside of a `BUILD_components.append` call